### ISSUE TITLE
MC 1.10.2 Crash, ArrayIndexOutOfBoundsException, SimplexOctave.noise

### ISSUE THREAD

**Issue by Zarepheth** [Team-RTG/Realistic-Terrain-Generation#995]
##### Affected Mod Versions:

(Please list other mods and their versions if you feel they are relevant to this issue. i.e. Biomes O' Plenty)
- Forge version:
  - forge 1.10.2-12.18.1.2095
- RTG version:
  - RTG-1.10.2-4.1.1.2
- Mods on stack trace:
  - Crystallogy
  - ExNihiloomnia
- Other major mods:
  - Biomes O'Plenty
  - Tinkers Construct
  - Weather 2
- World Seed:
  - 4937635207168846818
- Full list of mods in the attached crash report.
##### Issue Description:

Shortly after constructing a small dirt hut, a double chest, and emptying inventory into chest, was about to start digging downward when the game froze and crashed. Still on first day (or night), had been killed once by a panther near the starting location, and Weather 2 had a thunderstorm in progress at the time of the crash.
##### Extra Information:

[crash-2016-10-08_22.21.28-server.txt](https://github.com/Team-RTG/Realistic-Terrain-Generation/files/518083/crash-2016-10-08_22.21.28-server.txt)

**Comment by Zarepheth** [comment]
After filling out this bug report, I was able to re-enter the game at the point of the crash (give or take few seconds) and continue playing. I suffered another crash an hour or so later. The end point where the crash occurred is the same as the original crash report, but the stack trace is much, much, much longer and includes many more plugins.
- crystallogy
- quark
- harvestcraft
- techreborn
- jurassicraft

Considering the length of the stack trace and some repetitive patterns therein, I'm wondering if a stack-overflow could be the root cause (at least for the second crash)...

[crash-2016-10-08_23.49.08-server.txt](https://github.com/Team-RTG/Realistic-Terrain-Generation/files/518543/crash-2016-10-08_23.49.08-server.txt)

I've updated Forge and a couple of the mods and will resume playing...
